README.md

```diff
@@ -65,25 +121,25 @@ touch .secrets
   - Access the rails console for debugging
 
     ```bash
-    rails c
+    bundle exec rails c
     ```
 
   - Run the tests, excluding those that require the Yale VPN (the tilda(~) means the tag is excluded)
 
     ```bash
-    rspec --tag ~vpn_only:true
+    bundle exec rspec --tag ~vpn_only:true
     ```
 
   - Run only the tests that require the Yale VPN
 
     ```bash
-    rspec --tag vpn_only:true
+    bundle exec rspec --tag vpn_only:true
     ```
 
   - Run Rubocop to fix any style errors
 
     ```bash
-    rubocop -a
+    bundle exec rubocop -a
```

Comment:
undo this change (if you recommend cam sh it deals with the bundle exec for you)